cmd/podman/play/kube.go

```diff
@@ -92,21 +92,29 @@ func kube(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	for _, l := range report.Logs {
-		fmt.Fprintf(os.Stderr, l)
+	for _, pod := range report.Pods {
+		for _, l := range pod.Logs {
+			fmt.Fprintf(os.Stderr, l)
+		}
 	}
 
-	fmt.Printf("Pod:\n%s\n", report.Pod)
-	switch len(report.Containers) {
-	case 0:
-		return nil
-	case 1:
-		fmt.Printf("Container:\n")
-	default:
-		fmt.Printf("Containers:\n")
-	}
-	for _, ctr := range report.Containers {
-		fmt.Println(ctr)
+	for _, pod := range report.Pods {
+		fmt.Printf("Pod:\n")
+		fmt.Println(pod.ID)
+
+		switch len(pod.Containers) {
+		case 0:
+			return nil
```

Comment:
Should this be a `continue` so we print all pods in the case that there are more than 1?